lintreview/tools/py3k.py

```diff
@@ -1,66 +1,38 @@
 from __future__ import absolute_import
-import os
+
 import logging
+
 import lintreview.docker as docker
-from lintreview.tools import Tool, process_quickfix, stringify
+from lintreview.tools import stringify
+from lintreview.tools.pylint import Pylint
 
 log = logging.getLogger(__name__)
 
 
-class Py3k(Tool):
+class Py3k(Pylint):
     """
     $ pylint --py3k is a special mode for porting to python 3 which
     disables other pylint checkers.
     see https://github.com/PyCQA/pylint/issues/761
     """
 
     name = 'py3k'
+    accepted_options = ('ignore', )
 
     def check_dependencies(self):
         """
         See if python image is available
         """
         return docker.image_exists('python2')
 
-    def match_file(self, filename):
-        base = os.path.basename(filename)
-        name, ext = os.path.splitext(base)
-        return ext == '.py'
-
-    def process_files(self, files):
-        """
-        Run code checks with pylint --py3k.
-        Only a single process is made for all files
-        to save resources.
-        """
-        log.debug('Processing %s files with %s', files, self.name)
-        command = self.make_command(files)
-        output = docker.run('python2', command, self.base_path)
-        if not output:
-            log.debug('No py3k errors found.')
-            return False
-
-        output = output.split("\n")
-        output = [line for line in output if not line.startswith("*********")]
+    def make_command(self, files):
+        self.check_options()
```

Comment:
The py3k tool needs to remove the `python` option as `pylint --py3k` doesn't do anything useful in python3.

Reply:
Good catch, updated here: https://github.com/markstory/lint-review/pull/251/commits/61d75fb10ae3fa52eab41b8e62f436cf1a995bef